tests/test_port_dpb_acl.py

```diff
@@ -0,0 +1,126 @@
+        dvs.verify_acl_group_num(1)
+        dvs.verify_acl_group_member(acl_group_ids[0], acl_table_ids[0])
+        dvs.verify_acl_group_member(acl_group_ids[0], acl_table_ids[1])
+        dvs.verify_acl_port_binding(bind_ports)
+
+        #Delete ACL tables
+        dvs.remove_acl_table("test")
+        time.sleep(2)
+        dvs.verify_acl_group_num(1)
+        dvs.remove_acl_table("test1")
+        time.sleep(2)
+        dvs.verify_acl_group_num(0)
+    '''
+    @pytest.mark.skip()
+    '''
+    def test_one_acl_table_many_ports(self, dvs):
+        dvs.setup_db()
+
+        # Create ACL table and bind it to Ethernet0 and Ethernet4
+        bind_ports = ["Ethernet0", "Ethernet4"]
+        dvs.create_acl_table("test", "L3", bind_ports)
+        time.sleep(2)
+        acl_table_ids = dvs.get_acl_table_ids()
+        assert len(acl_table_ids) == 1
+        dvs.verify_acl_group_num(2)
+        acl_group_ids = dvs.get_acl_group_ids()
+        dvs.verify_acl_group_member(acl_group_ids[0], acl_table_ids[0])
+        dvs.verify_acl_group_member(acl_group_ids[1], acl_table_ids[0])
+        dvs.verify_acl_port_binding(bind_ports)
+
+        # Update bind list and verify
+        bind_ports = ["Ethernet4"]
+        dvs.create_acl_table("test", "L3", bind_ports)
+        time.sleep(2)
+        dvs.verify_acl_group_num(1)
+        acl_group_ids = dvs.get_acl_group_ids()
+        dvs.verify_acl_group_member(acl_group_ids[0], acl_table_ids[0])
+        dvs.verify_acl_port_binding(bind_ports)
+
+        # Breakout Ethernet0
+        dpb = DPB()
+        dpb.breakout(dvs, "Ethernet0", 4)
+        time.sleep(2)
+
+        #Update bind list and verify
+        bind_ports = ["Ethernet0", "Ethernet1", "Ethernet2", "Ethernet3","Ethernet4"]
+        dvs.create_acl_table("test", "L3", bind_ports)
+        time.sleep(2)
+        dvs.verify_acl_group_num(5)
+        acl_group_ids = dvs.get_acl_group_ids()
+        dvs.verify_acl_group_member(acl_group_ids[0], acl_table_ids[0])
+        dvs.verify_acl_group_member(acl_group_ids[1], acl_table_ids[0])
+        dvs.verify_acl_group_member(acl_group_ids[2], acl_table_ids[0])
+        dvs.verify_acl_group_member(acl_group_ids[3], acl_table_ids[0])
+        dvs.verify_acl_group_member(acl_group_ids[4], acl_table_ids[0])
+        dvs.verify_acl_port_binding(bind_ports)
+        time.sleep(2)
+
+        # Update bind list and verify
+        bind_ports = ["Ethernet4"]
+        dvs.create_acl_table("test", "L3", bind_ports)
+        dvs.verify_acl_group_num(1)
+        acl_group_ids = dvs.get_acl_group_ids()
+        dvs.verify_acl_group_member(acl_group_ids[0], acl_table_ids[0])
+        dvs.verify_acl_port_binding(bind_ports)
+
+        #Breakin Ethernet0, 1, 2, 3
+        dpb.breakin(dvs, ["Ethernet0", "Ethernet1", "Ethernet2", "Ethernet3"])
+        time.sleep(2)
+
+        # Update bind list and verify
+        bind_ports = ["Ethernet0", "Ethernet4"]
+        dvs.create_acl_table("test", "L3", bind_ports)
```

Comment:
write a new update_acl_table() and to update only binding ports.

Reply:
Done!